### ISSUE TITLE
[Feature Request] Enable "LongStrip Mode"/Remember zoom+auto scroll to the top of the image

### ISSUE THREAD

**Issue by GreenLandisaLie** [NO-ob/LoliSnatcher_Droid#104]
Reading manga in landscape mode with gallery scroll direction set to 'vertical' is great, I double click on the opened image to let it fit the entire screen then I scroll up/down as I'm reading and then the app loads the next image. That's when the problem happens - lolisnatcher doesn't remember that the previous image was zoomed and it will load every new image 'unzoomed' - forcing me to double tap and scroll to the top of the image every time I jump to a new page.

Possible solutions:
- An entirely new feature for "LongStrip Mode" - which would be ideal, many other apps do this.
OR (lazy solution:)
- Add new option that - when toggled ON and gallery scroll direction is set to 'vertical' - will automatically execute the "double tap function (zoom)" and then automatically scroll to the top of the image, each time a new image is loaded.

**Comment by NANI-SORE** [comment]
Well, LS is not meant to be a manga reader, since every next image you scroll to has a high chance of having completely different proportions compared to the previous one, therfore keeping zoom value consistent is not possible.
But I'll consider implementing longstrip option when I'll get around to improving media viewer somewhere down the line.

**Comment by GreenLandisaLie** [comment]
I agree 100%. In my case, the only time I ever felt the need for this was when trying to view manga through the Hydrus API. Although I'm aware that sankaku also allows the upload of doujinshis and mangas - to some extent and its possible some boorus will do the same in the future.
